packages/beacon-node/src/api/impl/validator/index.ts

```diff
@@ -912,7 +912,7 @@ export function getValidatorApi({
       // TODO: Add a flag to just send 0x00 as pubkeys since the Lodestar validator does not need them.
       const pubkeys = getPubkeysForIndices(state.validators, indexes);
 
-      const startSlot = computeStartSlotAtEpoch(stateEpoch);
+      const startSlot = computeStartSlotAtEpoch(epoch);
```

Comment:
This is the fix, right? Maybe it would be worth adding a comment clarifying what is done here?

Reply:
The previous code would have deserved a comment on why we use the stateEpoch instead of actual requested epoch which it obviously didn't. But this makes sense now, shouldn't require a comment imo